.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: dev-deploy
+
+on:
+  push:
+    branches:
+      - development
+  workflow_dispatch:
```

Comment:
Do we want to setup similar things for staging and prod?

Reply:
I think that exists above in `deploy.yml`, as far as I can tell. Would it be beneficial then to combine them and make `environ` a variable?

Reply:
Staging and prod setup are already included in the deploy.yml file.  For debug purpose we separate development out so we may test/debug any changes push to development brunch first instead of master brunch.

Reply:
This seems worthwhile to merge and road test in development. If we want to consolidate in the future, we can do so.